### ISSUE TITLE
Indicating that  package is ESM, not classic ES

### ISSUE THREAD

**Issue by Pomax** [cdnjs/packages#1776]
It looks like https://github.com/cdnjs/packages/blob/master/CONTRIBUTING.md does not cover the syntax necessary to indicate whether a package is an ES module, which means that the cdnjs page for module packages will give people the wrong code to use. For example, https://cdnjs.com/libraries/graphics-element provides people with

```html
<script src="https://cdnjs.cloudflare.com/ajax/libs/graphics-element/1.11.1/graphics-element.js" integrity="sha512-fz6xdkeMImCEkyHV+HPpg7C9cxoqc0w9RdSzm9ufj2wjFQFz71iJvyazoM9tXRGBzN40GuNY2e5fmimRiBFoJw==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
```
but if they put that on their page, they'll get a dev console error because the code they should have actually used is:

```html
<script type="module" src="https://cdnjs.cloudflare.com/ajax/libs/graphics-element/1.11.1/graphics-element.js" integrity="sha512-fz6xdkeMImCEkyHV+HPpg7C9cxoqc0w9RdSzm9ufj2wjFQFz71iJvyazoM9tXRGBzN40GuNY2e5fmimRiBFoJw==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
```

with an explicit `type="module" so that the browser can correctly load the lirbary. Can the contributor docs be updated to explain how to mark a library as ESM, so that the cdnjs website can then tap into that information and generate the correct HTML code?

**Comment by MattIPv4** [comment]
It is not currently possible to do so, hence it is not documented. I suspect this will need some work across the tools repo as well as the API + website to make happen, and then finally a docs update here and updates to any packages that need it.

**Comment by Pomax** [comment]
Noted - is there a better place to file this as an overarching issue? (since ESM isn't exactly new at this point, and the sooner the work starts, the sooner it's done, so the sooner people relying on cdnjs benefit =)

**Comment by MattIPv4** [comment]
This repo is probably fine 👍 Hopefully someone in the community will be interested in contributing the required changes

**Comment by Pomax** [comment]
Indeed! (but hopefully it also goes pretty high up on the cdnjs triage board)
